lib/hawkular/operations/operations_api.rb

```diff
@@ -326,12 +326,15 @@ def add_jdbc_driver(hash, &callback)
     # Exports the JDR report
     #
     # @param [String] resource_path canonical path of the WildFly server
+    # @param [Boolean] delete_from_server specifies whether the temporary file at the remote server should be deleted.
```

Comment:
I'm wondering if it's necessary to call it delete_from_server. Why simply "delete" is not enough? maybe I missed some additional use case?